avocado/core/job.py

```diff
@@ -310,7 +311,7 @@ def _make_test_suite(self, references=None):
 
         if not getattr(self.args, "dry_run", False):
             return suite
-        for i in xrange(len(suite)):
+        for i in range(len(suite)):
```

Comment:
Now there comes the question. If we are switching from py2 to py3, I'd prefer to sacrifice small perf regression in py2 execution to cleaner code and just used `range`. @clebergnu, @apahim what do you think?

Reply:
You mean dropping the `from six.moves import ...` line? We already have other six imports, so I'm  fine with how this is.